### ISSUE TITLE
Problem of input with the phone number fields

### ISSUE THREAD

**Issue by zuluzaire** [litomba/www#149]
On the ad creation page and on the on-boarding page, the using chrome on the Galaxy S7 mobile, when we input  the phone number there is a problem with placement of the cursor.
Check video on slack:

**Comment by AntonioVdlC** [comment]
Actually, it might be that I didn't update the phone input in the On Boarding page ... my bad! 😅

**Comment by zuluzaire** [comment]
No... still aint working neither on ad creation  nor on profil page.
